gnomad/variant_qc/evaluation.py

```diff
@@ -242,7 +170,7 @@ def compute_grouped_binned_ht(
     bin_ht: hl.Table, checkpoint_path: Optional[str] = None,
 ) -> hl.GroupedTable:
     """
-    Groups a Table that has been annotated with bins based on quantiles (`compute_quantile_bin` or
+    Groups a Table that has been annotated with bins based on quantiles (`compute_ranked_bin` or
```

Comment:
This function needs to be adjusted since `bin_stats` no longer exists as a global annotation (e.g., line 191). And does it still make sense to name the annotation on line 188 `quantile_bins`, now that we're not using a quantile approach? Might be less confusing to name it something else

Reply:
Thank you, will make changes now